toolchain/sem_ir/typed_insts.h

```diff
@@ -737,18 +737,28 @@ struct VarStorage {
   NameId name_id;
 };
 
+// These concepts are an implementation detail of the library, not public API.
+namespace Detail {
```

Comment:
Just flagging the other place to do s/Detail/Internal/

Reply:
Acknowledged.